src/index.html

```diff
@@ -10,13 +10,98 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="menu">
+        <ul>
+          <li>
+            <a
+              class="nav_link is-active"
+              href="#apple"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav_link"
+              href="#samsung"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav_link"
+              href="#smartphones"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              data-qa="hover"
+              class="nav_link"
+              href="#laptops&computers"
```

Comment:
Don't use spaces in the `<a>` tag's `href` property. The value `href="#laptops&computers"` should not contain `&` or spaces. If you want to link to two different sections, use two different links or a valid id.